src/semantic-router/pkg/observability/logging.go

```diff
@@ -0,0 +1,28 @@
+package observability
+
+import (
+	"encoding/json"
+	"log"
```

Comment:
maybe this is time to pick a logging pkg, e.g. https://betterstack.com/community/guides/logging/best-golang-logging-libraries/

Reply:
I prefer zap, but I'll create an issue to see what others think.

Reply:
this one https://github.com/vllm-project/semantic-router/issues/76